src/target/target_kind.cc

```diff
@@ -219,6 +219,7 @@ TVM_REGISTER_TARGET_KIND("llvm", kDLCPU)
     .add_attr_option<Bool>("system-lib")
     .add_attr_option<String>("runtime")
     .add_attr_option<Bool>("link-params", Bool(false))
+    .add_attr_option<Target>("host")
```

Comment:
No need to add them separately, you may add this line to the `TVM_REGISTER_TARGET_KIND ` macro